lua/nvim-treesitter/configs.lua

```diff
@@ -184,32 +235,63 @@ function M.setup(user_data)
 
   for mod, data in pairs(user_data) do
     if config.modules[mod] then
-      if type(data.enable) == 'boolean' then
-        config.modules[mod].enable = data.enable
-      end
-      if type(data.disable) == 'table' then
-        config.modules[mod].disable = data.disable
-      end
-      if config.modules[mod].keymaps and type(data.keymaps) == 'table' then
-        for f, map in pairs(data.keymaps) do
-          if config.modules[mod].keymaps[f] then
-            config.modules[mod].keymaps[f] = map
-          end
-        end
-      end
+      M.setup_module(config.modules[mod], data)
```

Comment:
Don't you want to use `get_module` here too for consistency ?

Reply:
Actual this processes the root modules... I think I should move this to the `setup_module` function.